.circleci/config.yml

```diff
@@ -159,9 +157,103 @@ jobs:
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
 
+  integration_tests:
+    executor: main-executor
+    environment:
+      COVERAGE: true
+    steps:
+      - checkout_and_fetch
+      - restore_node_modules
+      - restore_built_packages
+      - jq/install
+      - run:
+          name: 'Check if PR is approved'
+          command: |
+            PR_NUMBER=${CIRCLE_PULL_REQUEST##*/}
+            PR_APPROVAL_STATUS=$(curl -s https://api.github.com/repos/webex/webex-js-sdk/pulls/${PR_NUMBER}/reviews | jq ".[0].state")
+
+            if [ "$PR_APPROVAL_STATUS" -ne "APPROVED" ]; then
+              exit 1;
+            fi;
+      - run:
+          name: 'Generate Build Number'
+          command: echo "export BUILD_NUMBER=CircleCI-${CIRCLE_PR_USERNAME}-${CIRCLE_PR_REPONAME}-${CIRCLE_PR_NUMBER}_${CIRCLE_WORKFLOW_ID}" >> $BASH_ENV
+      - run:
+          name: 'Run all integration tests on Mac and Windows version of Chrome and Firefox'
+          command: npm run test -- --os Mac Windows --browsers Chrome Firefox
+      - run:
+          name: 'Check if any integration tests failed'
+          command: |
+            FAILURES=$(find reports -iname 'webex*.xml' -print0 | xargs -0 sed -n 's/.*failures=\"\([^\"]*\).*/\1/p' | xargs)
+            ERRORS=$(find reports -iname 'webex*.xml' -print0 | xargs -0 sed -n 's/.*errors=\"\([^\"]*\).*/\1/p' | xargs)
+
+            echo Failures $FAILURES
+            echo Errors $ERRORS
+
+            if [[ "$FAILURES" -ne *0* ]] || [[ "$ERRORS" -ne *0* ]]; then
+              echo 'Unit tests failed'
+              exit 1
+            fi
+      - store_test_results:
+          path: reports/
+      - store_artifacts:
+          path: reports/junit
+          destination: junit
+
+  version_bump:
+    executor: main-executor
+    steps:
+      - checkout_and_fetch
+      - restore_node_modules
+      - restore_built_packages
+      - skip_check
```

Comment:
What benefit does this give us? I don’t see anything requiring it to progress.

Reply:
This may have been find-and-replace job
And maybe for a future "this is where the skip job would go"
But all this does is run a git diff so it's pointless in this job's context